src/npc/emotes.py

```diff
@@ -0,0 +1,305 @@
+
+            draw_aa_line(self._image, center_pos, thickness, length, deg, (170, 121, 89))
+
+            # pygame.draw.line(self.emote_image, (170, 121, 89), start_pos, end_pos, thickness)
+
+            # increase degree by half the distance to the next emote,
+            #  to get the center of the current emote in the selector wheel
+            deg = math.pi * 2 * (i + .5) / len(self.emotes) - math.pi / 2
+
+            # blit first image of the emote as preview onto the selector wheel
+            self._image.blit(self.emote_manager.emotes[self.emotes[i]][0],
+                             (self.outer_radius - 18 * 2 + math.cos(deg) * self.center,
+                              self.outer_radius - 18 * 2 + math.sin(deg) * self.center))
+
+        # draw emote wheel outlines
+        pygame.draw.aacircle(self._image, (170, 121, 89), (self.outer_radius, self.outer_radius),
+                             self.inner_radius, 4)
+        pygame.draw.aacircle(self._image, (170, 121, 89), (self.outer_radius, self.outer_radius),
+                             self.outer_radius - 1, 4)
+
+        self.image = self._image.copy()
+
+        self.rect = pygame.Rect((0, 0), self.image.size)
+        self.pos = self.rect.topleft
+
+    @property
+    def pos(self):
+        return self._pos
+
+    @pos.setter
+    def pos(self, value):
+        self._pos = value
+        self.rect.update((self._pos[0] - self.rect.width / 2,
+                          self._pos[1] - self.rect.height / 2), self.rect.size)
+
+    def update(self, *args, **kwargs):
+        if self.image.get_alpha() == 0 or self._last_emote_index == self.emote_index:
+            return
+
+        self._last_emote_index = self.emote_index
+        self.current_emote = self.emotes[self.emote_index % 8]
+
+        self.image = self._image.copy()
+
+        current_emote_index = self.emote_index % 8
+
+        # draw thicker and brighter lines around the currently selected emote
+        deg = math.pi * 2 * current_emote_index / 8 - math.pi / 2
+
+        # center_pos and length had to be slightly adjusted to be neither to short,
+        #  nor to extend beyond the edge of the selector wheel
+        center_pos = (self.outer_radius + math.cos(deg) * (self.center - 3),
+                      self.outer_radius + math.sin(deg) * (self.center - 3))
+        thickness = 6
+        length = self.outer_radius - self.inner_radius + 5
+
+        draw_aa_line(
+            self.image, center_pos, thickness, length, deg, (243, 229, 194)
+        )
+
+        deg = math.pi * 2 * (current_emote_index + 1) / 8 - math.pi / 2
+        center_pos = (self.outer_radius + math.cos(deg) * (self.center - 3),
+                      self.outer_radius + math.sin(deg) * (self.center - 3))
+
+        draw_aa_line(
+            self.image, center_pos, thickness, length, deg, (243, 229, 194)
+        )
+
+        start_deg = math.pi * 2 * -current_emote_index / 8 + math.pi / 4
```

Comment:
For some reason pygame.draw.arc doesn't seem to work with the same degrees, hence these shenanigans. Please tell me what I've done wrong 😓

Reply:
It may be because you're actually trying to pass values in degrees, while `pygame.draw.arc` expects values in radians

Reply:
Nah, I just named the variables _deg because that sounded most fitting to me off the top of my head 😅. The value is definitely in radians, so I should probably rename them...